client/src/lib/session/components/Slides/Slides/Blocks/Video.tsx

```diff
@@ -106,9 +108,10 @@ const Video: React.FC<VideoProps> = ({
     // Check that the progressRef is not set back to zero to trigger a reset
     if (!autoPlayLoop && !onEndRef.current) {
       onEndRef.current = true;
+      seek(0);
```

Comment:
I don't think this is what we want. Before the video just stopped when coming to an end. And then if you press the rewind button it started playing again as the playing state was already playing.